plugins/git-proxy-plugin-samples/package.json

```
@@ -16,6 +16,6 @@
     "express": "^4.18.2"
   },
   "peerDependencies": {
-    "@finos/git-proxy": "1.3.5-alpha.5"
+    "@finos/git-proxy": "1.4.0"
```

We don't want to pin the version explicitly. Let's use the [semver constraint](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept).
```suggestion
    "@finos/git-proxy": "^1.4.0"
```